app/mailers/digest_mailer.rb

```diff
@@ -1,17 +1,49 @@
-class DigestMailer < ActionMailer::Base
-  default from: ENV["MAIL_NOTIFICATION"]
+class DigestMailer < ApplicationMailer
 
-  # Subject can be set in your I18n file at config/locales/en.yml
-  # with the following lookup:
-  #
-  # en.digest_mailer.daily_digest.subject
+  def daily_digest(history, user_id)
```

Comment:
Unused method argument - `history`. If it's necessary, use `_` or `_history` as an argument name to indicate that it won't be used.